eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -118,18 +121,49 @@ object NodeParams {
       .withFallback(ConfigFactory.parseFile(new File(datadir, "eclair.conf")))
       .withFallback(ConfigFactory.load())
 
-  def getSeed(datadir: File): ByteVector = {
-    val seedPath = new File(datadir, "seed.dat")
-    if (seedPath.exists()) {
-      ByteVector(Files.readAllBytes(seedPath.toPath))
-    } else {
-      datadir.mkdirs()
-      val seed = randomBytes32
-      Files.write(seedPath.toPath, seed.toArray)
-      seed
+  private def readSeedFromFile(seedPath: File): ByteVector = {
+    logger.info(s"use seed file: ${seedPath.getCanonicalPath}")
+    ByteVector(Files.readAllBytes(seedPath.toPath))
+  }
+
+  private def writeSeedToFile(path: File, seed: ByteVector): Unit = {
+    Files.write(path.toPath, seed.toArray)
+    logger.info(s"create new seed file: ${path.getCanonicalPath}")
+  }
+
+  private def migrateSeedFile(old: File, `new`: File): Unit = {
+    if (old.exists() && !`new`.exists()) {
+      Files.copy(old.toPath, `new`.toPath)
+      logger.info(s"migrate seed file: ${old.getCanonicalPath} → ${`new`.getCanonicalPath}")
     }
   }
 
+  def getSeeds(datadir: File): Seeds = {
+    // Previously we used one seed file ("seed.dat") to generate the node and the channel private keys
+    // Now we use two separate files and thus we need to migrate the old seed file if necessary
+    val oldSeedPath = new File(datadir, "seed.dat")
+    val nodeSeedFilename: String = "nodeSeed.dat"
+    val channelSeedFilename: String = "channelSeed.dat"
```

Comment:
I don't know why, but camel case doesn't work well for files.

```suggestion
    val nodeSeedFilename: String = "node_seed.dat"
    val channelSeedFilename: String = "channel_seed.dat"
```

Reply:
Done